packages/remote-config/src/definitions.ts

```diff
@@ -45,6 +45,15 @@ export interface FirebaseRemoteConfigPlugin {
   setMinimumFetchInterval(
     options: SetMinimumFetchIntervalOptions,
   ): Promise<void>;
+  /**
+   * Set the remote config settings.
+   * 
+   * On Android, the settings values are saved in SharedPreferences.
+   * Therefore, please note that even if setSettings is removed, the saved values will still be used.
```

Comment:
```suggestion
   * On Android, the settings values are persisted in SharedPreferences.
```

That should be enough.